controllers/falcon/falconnodesensor_controller.go

```diff
@@ -683,7 +683,7 @@ func (r *FalconNodeSensorReconciler) conditionsUpdate(condType string, status me
 	})
 
 	if err := r.Status().Update(ctx, nodesensor); err != nil {
-		logger.Error(err, "Failed to update FalconNodeSensor status")
+		logger.Error(err, "Failed to update FalconNodeSensor status", "Failed to update the Condition at Reasong", reason)
```

Comment:
Reasong ❓❗

Reply:
HAHAHA.... I just wanted a "Resong" for you to comment. ;-) 

Fixed.